pkg/network/ebpf/c/protocols/kafka/kafka-parsing.h

```diff
@@ -12,6 +12,7 @@ static __always_inline bool kafka_allow_packet(skb_info_t *skb_info);
 static __always_inline bool kafka_process(conn_tuple_t *tup, kafka_info_t *kafka, pktbuf_t pkt, kafka_telemetry_t *kafka_tel);
 static __always_inline bool kafka_process_response(void *ctx, conn_tuple_t *tup, kafka_info_t *kafka, pktbuf_t pkt, skb_info_t *skb_info);
 static __always_inline void update_topic_name_size_telemetry(kafka_telemetry_t *kafka_tel, __u64 size);
+static __always_inline void update_classified_api_version_hits_telemetry(kafka_telemetry_t *kafka_tel, s16 api_key, s16 api_version);
```

Comment:
as we care only for produce (api 0) and fetch (api 1), and the versions are 0-12 to produce, and 0-17 to consume, I'm think we should be using `__u8` for `api_key` and `api_version`